systemtests/tests/python-bareos-test/python-bareos-unittest.py

```diff
@@ -640,6 +640,31 @@ def _test_list_with_invalid_jobid(self, director, jobid):
                 u'Command {} should not return results. Current result: {} visible'.
                 format(listcmd, str(result)))
 
+    def search_joblog(self, director, jobId, patterns):
+
+        if isinstance(patterns, list):
+            pattern_dict = { i: False for i in patterns }
+        else:
+            pattern_dict = { patterns: False }
+
+        result = director.call('list joblog jobid={}'.format(jobId))
+        joblog_list = result['joblog']
+
+        found = False
+        for pattern in pattern_dict:
+            for logentry in joblog_list:
+                if re.search(pattern, logentry['logtext']):
+                    pattern_dict[pattern] = True
+            self.assertTrue(pattern_dict[pattern], 'Failed to find pattern "{}" in Job Log of Job {}.'.format(pattern, jobId))
```

Comment:
The test (also, as many others) relies on the fact that a job message in the source code has this expected pattern. During a code refactoring this is not always obvious. Therefore, would you like to add a comment to that part in the code to point out the fact?

Reply:
Thank you for your review. IMHO this bugs affects almost all users, as the CatalogBackup job of the Bareos default installation uses local RunScripts. Also the Bareos documentation delivers examples for Admin jobs for some tasks.

Reply:
Yes, I understand it should run on the director. 

As mentioned above, would you add an additional comment in the source code that mentions the dependency with the test?